azure_functions_worker/dispatcher.py

```diff
@@ -306,6 +310,13 @@ async def _handle__worker_init_request(self, request):
                 self.load_function_metadata(
                     worker_init_request.function_app_directory,
                     caller_info="worker_init_request")
+
+                if HttpV2Registry.http_v2_enabled():
+                    capabilities[constants.HTTP_URI] = \
+                        initialize_http_server(self._host)
+
+            except HttpServerInitError:
```

Comment:
Why an explicit raise if it is going to be caught in the next line itself?

Reply:
![image](https://github.com/Azure/azure-functions-python-worker/assets/12449837/3dd01710-8e2e-457b-97de-ae08027c2c38)
http server starting up failure shall be raised fail fast. if going into exception, it is captured as metadata exception and raised in metadata error response.

Reply:
also throwing it there make senses as capabilites wont be set in init/env_reload. there is no meaning to continue to handle_metadata_request